src/tests/checkout.spec.js

```diff
@@ -0,0 +1,62 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { login, password } = require('../credentials/credentials.json');
+const { firstName, lastName, zipCode } = require('../credentials/test_data.json');
+const { extractNumberFromStr, arrayOfRandomItemIndex } = require('../utils/methods');
+
+test.describe('Test three: Checkout page', () => {
+    test.beforeEach(async ({ loginPage, inventoryPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin(login, password);
+        await expect(inventoryPage.headerTitle).toBeVisible();
+    });
+
+    test('Proceed to Checkout, place an Order and verify products', async ({
+        inventoryPage, shopingCartPage, checkoutFirstPage, checkoutSecondPage,
+    }) => {
+        // Perform adding random quantity of random items to the Shopping Cart
+        const unicRandomIndex = arrayOfRandomItemIndex(await inventoryPage.getNumberOfItems());
+        for (let i = 0; i < unicRandomIndex.length; i += 1) {
+            await inventoryPage.addItemToCartById(unicRandomIndex[i]);
+        }
+
+        // Check if all products were added to the Shopping Cart
+        const addedItemsInCart = await inventoryPage.getNumberOfItemsInCart();
+        await expect(inventoryPage.shopingCartBadge).toHaveText(addedItemsInCart);
+
+        // Go to Shopping Cart and verify page
+        await inventoryPage.shopingCart.click();
+        await expect(shopingCartPage.headerTitle).toBeVisible();
+
+        // Collect information of added items in shopping cart to verify details on Checout later
+        const allItemInfoInCart = await shopingCartPage.collectedItems();
```

Comment:
for consistency, I'd collect items from inventory page (only those I'm going to add to cart) and add additional comparison checking those items are in cart and then on checkout